src/lib/utils/options/readers/tsconfig.ts

```diff
@@ -26,6 +26,30 @@ export class TSConfigReader extends OptionsComponent
     private static OPTIONS_KEY:string = 'tsconfig';
 
 
+    private readConfigFile(event:DiscoverEvent, fileName:string) {
+        if (!FS.existsSync(fileName)) {
```

Comment:
As I get it we need the *raw* compiler options. `parseJsonConfigFileContent()` does follow the `extends` chain and aggregate the data but not in the *raw form*. If we use the `options: CompilerOptions` property from the result of `parseJsonConfigFileContent()` we will have to remove all internal options and map the enum represented ones(`ModuleKind`, `ScriptTarget`, `NewLineKind`, ...) to the correct string values.